docs/api-reference/account_filter.md

```diff
@@ -2,7 +2,7 @@
 sidebar_position: 4
 ---
 
-# Account Filter
+# `AccountFilter`
```

Comment:
The url for this page is 

```
http://localhost:3000/api-reference/account_filter
```

underscores in URLS are a poor taste, because the urls are styled with an underline anyway. 

So, we need to change _something_ here. 

I don't know what naming scheme would be best. 

```
http://localhost:3000/api-reference/account-filter
```

sounds ok to me